File: market/test/java/ru/yandex/market/b2b/clients/postgres/DatabaseTest.java
package ru.yandex.market.b2b.clients.postgres;

import org.junit.jupiter.api.Test;

import ru.yandex.market.javaframework.postgres.test.AbstractJdbcRecipeTest;

public class DatabaseTest extends AbstractJdbcRecipeTest {

    @Test
    public void simpleTest() {
        jdbcTemplate.execute("SELECT 1");
    }
}

File: market/test/kotlin/ru/yandex/market/rom/postgres/DatabaseTest.kt
package ru.yandex.market.rom.postgres

import org.junit.jupiter.api.Test
import ru.yandex.market.javaframework.postgres.test.AbstractJdbcRecipeTest

class DatabaseTest : AbstractJdbcRecipeTest() {
    @Test
    fun simpleTest() {
        jdbcTemplate.execute("SELECT 1")
    }
}
